### DISCUSSION TITLE
Firmware Downgrade keine Verbindung mehr mit Admin-AP möglich

### DISCUSSION THREAD

**Discussion by MLG1087** [tbnobody/OpenDTU#1964] [Q&A]
Hallo,
ich habe Aufgrund von Verbindungsproblemen mit meinem Wechselrichter mit der Firmware herumgespielt und ein Downgrade von der v24.2.12 auf die  v23.11.15 vorgenommen. Nun verbindet sich die DTU nicht mehr mit meinem WIFI und auf den Admin-AP der DTU kann ich nicht mehr zugreifen bzw sagt der PC/Smartphone das WIFI-Passwort ist falsch. Ich habe a Passwort (Login oder AP) keine Änderungen vorgenommen. 

Wie kann ich einen Reset der DTU machen oder das ding neu flashen?

**Comment by ghost** [comment]


**Reply by MLG1087** [reply]
Vielen Dank für den Tipp! Das hat für mich geklappt :D

**Reply by MLG1087** [reply]
Mir fehlt nun das Device-Profile für die Hoymiles HMS Serie, welche ich zuvor pre-installed hatte. @schneeer Hast du noch einen Tipp woher ich das bekomme und wie das funktioniert?

**Reply by MLG1087** [reply]
So ein Profile Generator ist bestimmt der Hammer für Leute wie mich :D
Ich habe die selbstbau-pv-de Hardware. Die Komponenten drauf heißen 900M20S, ESP-WROOM-32, ML01DP5.

**Reply by MLG1087** [reply]
[esp-web-tools-logs.txt](https://github.com/tbnobody/OpenDTU/files/15214056/esp-web-tools-logs.txt)

Jetzt wollte ich soeben das nodemcu_esp32.json ausprobieren und stecke die DTU wieder an und sie startet offensichtlich nicht. Die LED auf dem Board blinkt wie verrückt und der beigefügte LOG wird ausgespuckt. Das flashen mit dem community-flasher hat leider nicht geklappt. Bitte nochmal um Hilfe.

**Reply by MLG1087** [reply]
Vielen Dank für die Recherche! 
Ich habe nun zuerst Flash-Download-Tool wie von selbstbau-pv.de geflashed. Nachdem es noch nicht funktioniert hat und der gleiche Fehler wieder gekommen ist habe ich mit dem ESP-Tool den erase gemacht und nochmal mit dem Download-Tool geflashed.

Jetzt erreiche ich die DTU wieder :D juhu. Das Profile hat auch geklappt. 
Nun habe ich nur noch das Problem, dass er die Verbindung zum Wechselrichter immer wieder verliert. Aber das scheint ein Bug zu sein, da es mehrere Leute haben.

Danke nochmal für deine Hilfe!

**Reply by MLG1087** [reply]
Ich hatte beide schon direkt 1-5 Meter Luftlinie entfernt aufgestellt und hatte das gleiche. 
Morgen habe ich direkt für wenige Minuten eine Verbindung wo der Wechselrichter die Daten sendet (oder wenn ich ihn neu starte). Danach scheint er dann in der DTU rot auf und die Daten werden nicht mehr geliefert. 

Das Spannende: Daten kommen keine an aber neu Starten oder ein Limit per Hand kann ich über die DTU setzen.

Ändern der Seriennummer oder der Sendeleistung macht keinen Unterschied.

**Comment by ghost** [comment]


**Reply by MLG1087** [reply]
Die Elkos sind in der Erwähnten größe drin. Ich habe ein anderes Netzteil nun verwendet und es sieht Spitze aus. Die DTU hat nun seit mehr als zwei Stunden durchgehend alle 5 Sekunden ein Update erhalten. Das hatte ich bisher noch nie.
